src/app/shared/components/map/map.component.ts

```diff
@@ -214,6 +219,27 @@ export class MapComponent implements AfterViewInit, OnDestroy {
     });
   }
 
+  setPrevWorkShopMarker() {
+    this.selectedWorkshop$.pipe(takeUntil(this.destroy$)).subscribe(value => {
+      if ('/workshop-details/'+value.id === this.previousUrl.getPreviousUrl()) {
+        const targetMarkers = this.workshopMarkers.filter((workshopMarker) => {
+          const {lat, lng} = workshopMarker.marker.getLatLng()
+          if ( lat === value.address.latitude && lng === value.address.longitude) {
+            return true
+          } else {
+            return false
+          }
```

Comment:
it's redundant. I think it will work this way:
return ( lat === value.address.latitude && lng === value.address.longitude);